### ISSUE TITLE
执行实验报chaosblade file not found

### ISSUE THREAD

**Issue by mCapricorn** [chaosblade-io/chaosblade-box#38]
**报错现象**
目标机器上已经安装并启动了chaosagent：
root     23505     1  0 Apr06 ?        00:00:11 /opt/chaos/chaosagent --namespace default --appInstance chaos-default-app --appGroup chaos-default-app-group --port 19527 --transport.endpoint xxx.xxx.xxx.xxx:8080
执行实验的时候，报错：chaosblade file not found

**有个疑问**
com.alibaba.chaosblade.box.service.task.stateless.DefaultActivityTaskPhaseHandler#handle 方法里：
L159：requestCommand.setPort(chaosAgentPort);
这里port为什么是设的chaosagent的端口，而不是blade server start时绑定的端口？难道说chaosagent也具有执行实验的能力？

ps：内网对上传图片有限制，就只能文字描述了，还请见谅。

**Comment by tiny-x** [comment]
chaosagent 调用 blade 执行演练的。chaosagent 调用 blade 默认路径是 /opt/chaosblade/blade , 如果不存在就会报那个错。手动安装探针的话，也要手动安装下工具到 /opt/chaosblade 路径下面。

**Comment by mCapricorn** [comment]
> chaosagent 调用 blade 执行演练的。chaosagent 调用 blade 默认路径是 /opt/chaosblade/blade , 如果不存在就会报那个错。手动安装探针的话，也要手动安装下工具到 /opt/chaosblade 路径下面。

OK，已解决，3q~

**Comment by racheldaguenethahn** [comment]
Hi, I also have this kind of error when I run a workflow.

- I deployed chaosblade-box-web through an Alibaba ECI.
- Into the web interface, ProbeManager menu, I added a new ssh connection to my ECS. On the ECS, the chaosblade agent is deployed in /opt/chaos/ directory
- I created a new Experiment just to create an empty file into ECS:/tmp directory.

When I run the experiment, I have this message

2022-01-12 09:47:24.627, INFO , [EXPERIMENT-TASK-THREAD-0] c.a.c.box.service.task.stateless.DefaultActivityTaskPhaseHandler - sub task executing，task id: 1481186035317559298，phase：ATTACK, sub task id: 1481186035334336513, machines: iZgw8crd3ah122318x9qu3Z-172.16.1.178, is success: false, fail message: chaosblade file not found

Must I add something somewhere?

Thanks in advance
Rachel
